src/SmartComponents/RunTaskModal/__tests__/RunTaskModal.tests.js

```diff
@@ -185,7 +185,7 @@ describe('RunTaskModal', () => {
       </Provider>
     );
 
-    const nextButton = screen.getByLabelText('next-button');
+    const nextButton = screen.getByLabelText('taska-next-button');
```

Comment:
`taska`?

Reply:
slug: 'taska', is whats passed in inside the props for this test